src/main/scala/ai/privado/languageEngine/ruby/passes/GlobalImportPass.scala

```diff
@@ -37,77 +36,25 @@ import io.shiftleft.semanticcpg.language.*
 
 import scala.collection.Seq
 
-class GlobalImportPass(cpg: Cpg, packageTable: PackageTable, globalSymbolTable: SymbolTable[LocalKey])
-    extends PrivadoSimpleCpgPass(cpg) {
-
-  /*
-  lazy val modules: Set[String] =
```

Comment:
nit: lets remove this and other commented code as well

Reply:
@tuxology This whole code is already removed 😄

Reply:
🤯 Need coffee ☕